locale/fi/LC_MESSAGES/django.po

```diff
@@ -303,50 +304,56 @@ msgstr ""
 "palautuskertojen raja on ylitetty. Pisteet tallennetaan, mutta ne eivät "
 "sisälly virallisiin tuloksiin."
 
-#: course/permissions.py:20 course/permissions.py:88
+#: course/permissions.py:20
+#, fuzzy
```

Comment:
`fuzzy` means that the translation is not active and it is waiting for the developer to fix it. The `manage.py makemessages` command adds fuzzy markers when there are strings that are similar to each other or for some other reason, the command can not automatically decide which strings are really used.

When you have fixed the strings, the `fuzzy` word and the lines starting with `#|` must be removed.

https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#localization-how-to-create-language-files
https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html
https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html

Reply:
The final translations and fixing of the po-files should be done when everything else is ready.